classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -492,12 +492,18 @@ function publishIssue($args, $request) {
 				'status' => STATUS_SCHEDULED,
 				'count' => 5000, // large upper limit
```

Comment:
Can you remove this `count` param? It used to be that the `getMany` imposed a default count but that is no longer the case.